### PR TITLE
Added utilization bar chart and fixed line chart IDs

### PR DESCRIPTION
**Pull request by dlabrecq**
Needed for new patternfly site.
@dtaylor113 @jeff-phillips-18 please review.

### CODE REVIEW

**Comment by dtaylor113** [comment]
LGTM

File: tests-src/_includes/widgets/charts/line-multiple.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
   var c3ChartDefaults = $().c3ChartDefaults();
   var lineChartConfig = c3ChartDefaults.getDefaultLineConfig();
-  lineChartConfig.bindto = '#lineChart';
+  lineChartConfig.bindto = '#{{ include.chart1 }}';
```
Comment (jeff-phillips-18) [inline tests-src/_includes/widgets/charts/line-multiple.html:16]: convention is no spaces around the vars

This is optional, sorry my mind went immediately to angular mode ;)

**Comment by jeff-phillips-18** [comment]
LGTM

**Comment by dlabrecq** [comment]
This has been duplicated in many places from Robb's original templates. I'll modify all of these instances at once, during the next update.

**Pull request merged**